File: 1832. Check if the Sentence Is Pangram/1832. Check if the Sentence Is Pangram.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        vector<int> alpha(26);

        for(char c: sentence){
            alpha[c-'a']++;
        }

        for(int i=0; i<26; i++){
            if(alpha[i]==0) return false;
        }

        return true;
    }
};

File: 1832. Check if the Sentence Is Pangram/1832. Check if the Sentence Is Pangram.cs
public class Solution {
    public bool CheckIfPangram(string sentence) {
        int[] alpha  = new int[26];

        foreach(char c in sentence){
            alpha[c-'a']++;
        }

        foreach(int i in alpha){
            if(i==0) return false;
        }
        return true;
    }
}
